jdaviz/core/marks.py

```diff
@@ -538,6 +542,7 @@ def __init__(self, viewer, x=[], y=[], **kwargs):
         self.viewer = viewer
         # color is same blue as import button
         kwargs.setdefault('colors', [accent_color])
+        self.label = kwargs.get('label')
```

Comment:
what uses this?

Reply:
I'm using the label throughout ramp extraction to identify marks created for single-pixel ramps generated from subsets. These marks [get a `label` with the subset name](https://github.com/spacetelescope/jdaviz/blob/99206ffb50a0031f3c0b9cefd96ec5b351645f1e/jdaviz/configs/rampviz/plugins/ramp_extraction/ramp_extraction.py#L176), which makes a natural/easy/readable way to update the marks when their subsets are [added](https://github.com/spacetelescope/jdaviz/blob/99206ffb50a0031f3c0b9cefd96ec5b351645f1e/jdaviz/configs/rampviz/plugins/ramp_extraction/ramp_extraction.py#L184), [updated](https://github.com/spacetelescope/jdaviz/blob/99206ffb50a0031f3c0b9cefd96ec5b351645f1e/jdaviz/configs/rampviz/plugins/ramp_extraction/ramp_extraction.py#L202-L210), [deleted](https://github.com/spacetelescope/jdaviz/blob/99206ffb50a0031f3c0b9cefd96ec5b351645f1e/jdaviz/configs/rampviz/plugins/ramp_extraction/ramp_extraction.py#L191).